subsys/net/lib/sockets/Kconfig

```diff
@@ -244,8 +244,7 @@ config NET_SOCKETS_CAN_RECEIVERS
 	  Socket-CAN interface.
 
 config NET_SOCKETPAIR
-	bool "Support for the socketpair syscall [EXPERIMENTAL]"
-	select EXPERIMENTAL
+	bool "Support for the socketpair syscall"
```

Comment:
I'd like @cfriedt to comment on that.

Reply:
Will handle that in an upcoming PR, but won't be ready for 3.3